src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
+  .menu-bars {
+    position: fixed;
+    left: calc(50% - 134px);
+    bottom: 8px;
+  }
+  .menu-bars--isAdmin {
+    left: calc(50% - 168px);
```

Comment:
This looks mysterious. Can you leave a comment here what the `50%-168px` are about?

Reply:
Basically you're moving the top left corner of the block to the middle of the screen and afterwards move it back half the width of the block. The end result is a centered block on the horizontal axis / width.